management-pki/src/main/scala/akka/pki/kubernetes/PemManagersProvider.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2017-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.pki.kubernetes
+
+import java.io.ByteArrayInputStream
+import java.io.File
+import java.nio.charset.Charset
+import java.nio.file.Files
+import java.security.KeyStore
+import java.security.PrivateKey
+import java.security.cert.Certificate
+import java.security.cert.CertificateFactory
+import java.security.cert.X509Certificate
+
+import scala.concurrent.blocking
+
+import akka.annotation.ApiMayChange
+import akka.pki.pem.DERPrivateKeyLoader
+import akka.pki.pem.PEMDecoder
+import javax.net.ssl.KeyManager
+import javax.net.ssl.KeyManagerFactory
+import javax.net.ssl.TrustManager
+import javax.net.ssl.TrustManagerFactory
+
+/**
+ * Convenience methods to ease building an SSLContext from k8s-provided PEM files.
+ */
+// Duplicate from https://github.com/akka/akka/blob/31f654768f86db68f4c22daa2cbd0bae28fc1fad/akka-remote/src/main/scala/akka/remote/artery/tcp/ssl/PemManagersProvider.scala#L35
+// Eventually that will be a bit more open and we can reuse the class from akka in akka-management.
+// See also https://github.com/akka/akka-http/issues/3772
+@ApiMayChange
+object PemManagersProvider {
```

Comment:
Is this class thought be used by users? If not maybe we should leave it private for now?